src/Components/AddNewProject/AddNewProject.tsx

```diff
@@ -0,0 +1,106 @@
+
+import axios from "axios";
+import { useContext, useState } from "react";
+import { useForm } from "react-hook-form";
+import { Link, useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import { AuthContext } from "../../Context/AuthContext";
+
+export default function AddNewProject() {
+
+    const navigate = useNavigate();
+    const {baseUrl , requstHeaders} : any = useContext(AuthContext);
+    const [isLoding , setIsLoding] = useState(false);
+
+
+    const {
+        register,
+        handleSubmit,
+        formState : { errors },
+    } = useForm();
+
+    // ****************** to add new project **********************
+    const onSubmit = (data)=> {
+        // console.log(data);
+        setIsLoding(true);
+
+        axios.post(`${baseUrl}/Project` , data ,
+        {
+            headers : requstHeaders,
+        })
+        .then((response)=> {
+            // console.log(response);
+            setIsLoding(false);
+            navigate("/dashboard/projects")
+            toast.success("Added Successfully")
```

Comment:
Consider adding the message coming from the backend and provide "Added Successfully" as an alternative message if no message is coming from the backend
response?.data?.message  ||"Added Successfully"

you need to check the repsonse , to know exactly where the message is.